packages/contracts-bedrock/scripts/testing/test-extra-fuzzy.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script is used to run extra fuzz test iterations on any fuzz tests that
+# have been newly added in a PR. We typically want to run extra fuzz iterations
+# when new tests are added to make sure that they are not flaky with some small
+# percentage of fuzz runs.
+
+# Set the number of fuzz runs to run
+# 350000 fuzz runs will guarantee that any test that fails 1% of the time with
+# the default 512 fuzz runs will fail 99.9% of the time (on average) inside of
+# this script.
+FUZZ_RUNS=350000
```

Comment:
small nit: this could be an argument with a default

Reply:
Changed to argument